pkg/server/backuprestoreserver.go

```diff
@@ -177,9 +178,9 @@ func (b *BackupRestoreServer) runServerWithSnapshotter(ctx context.Context, rest
 
 	go defragmentor.DefragDataPeriodically(ctx, b.config.EtcdConnectionConfig, b.defragmentationSchedule, ssr.TriggerFullSnapshot, b.logger)
 
-	//TODO @aaronfern: Add functionality for member garbage collection
 	if b.config.HealthConfig.MemberLeaseRenewalEnabled {
 		go heartbeat.RenewMemberLeasePeriodically(ctx, b.config.HealthConfig, b.logger, b.config.EtcdConnectionConfig)
+		go membergarbagecollector.RunMemberGarbageCollectorPeriodically(ctx, b.config.HealthConfig, b.logger, b.config.EtcdConnectionConfig)
```

Comment:
Are there any scenarios where we would like to have member lease updates but no member garbage collection or vice versa?
If yes then another flag makes sense, but if not then we could just rename the flag to something more appropriate

Reply:
the thing is `membergarbagecollector.RunMemberGarbageCollectorPeriodically()` will only start by `leading-sidecar` and leading-sidecar will also gonna stop it once it is no longer `backup-leader`.
And heartbeat.RenewMemberLeasePeriodically() will start by both `backup-follower` as well as `backup-leader`. 
So both have different non-overlapping usage.

Reply:
The reason for switching off periodic garbage collection of members is the same as for switching off member lease renewal, which is the backward-compatibility single-node scenario. I cannot think of any scenario of multi-node scenario where you might want to switch off either of them. It might make sense to keep separate flags as @ishan16696 mentioned but both will be switched on or off together.

Also, as @ishan16696 mentioned, it is better to start and stop the member garbage collector goroutine in the on the [start](https://github.com/gardener/etcd-backup-restore/pull/353/files#diff-df6f7abbd9069d0a874a18b5717de5b0fcd68dac32184fdcdc2a26019be2296fR188-R189) and [stop](https://github.com/gardener/etcd-backup-restore/pull/353/files#diff-df6f7abbd9069d0a874a18b5717de5b0fcd68dac32184fdcdc2a26019be2296fR192-R201) callback of backup leader election because it is supposed to be done only by the [leading backup sidecar](https://github.com/gardener/etcd-druid/blob/master/docs/proposals/multi-node/README.md#work-flows-only-on-the-leading-member).

Reply:
I think it is better to leave it like this for now once @aaronfern  rebased this PR over  [LeaderElection PR](https://github.com/gardener/etcd-backup-restore/pull/353) then we can decide whether it make sense to have separate flag or not.